templates/partials/records/record-sph-item.html

```diff
@@ -0,0 +1,61 @@
+<article class="sph-records__item" id="{{getinventorynumber details}}">
+  {{#if images}}
+  <a href="{{firstImage images 0 'large' }}" data-lightbox>
+    <figure class="sph-records__figure">
+      <img src="{{firstImage images 0 }}" class="sph-records__img" alt="{{this.title}}" />
+    </figure>
+  </a>
+  {{ else }}
+  <figure class="sph-records__figure">
```

Comment:
previously, I only showed this fallback figure if any of the children had an image. Now this would need to be calculated from various levels of children, again might be better calculated pre-handlebars?